### ISSUE TITLE
ProtoFile as collection

### ISSUE THREAD

**Issue by et1975** [jet/falanx#35]
**Is your feature request related to a problem? Please describe.**
Having multiple `ProtoFile` entries (or multiple results from `Include` specifying a pattern) currently results in the error:

> Could not find a part of the path 'file1.proto;file2.proto'.

**Describe the solution you'd like**
Handle the individual items in the collection instead of assuming it's a single file.

**Describe alternatives you've considered**
Project-per-proto might work, but I think it's too much overhead.

**Additional context**
n/a.

**Comment by et1975** [comment]
Just realized generating the files separately, instead of automagically as part of the build might work (and would be preferable) too, but it has another problem: I'd like to include the CLI tool in the same project, but building such a project currently fails unless there are are `ProtoFile` entries.

**Comment by 7sharp9** [comment]
One one file is supported as input to Falanx, the import statement should be used for this purpose.  Froto does not yet support import although there is beta support for recursing generation on imports

**Comment by enricosada** [comment]
Yes, atm multiple proto in fsproj are not support yet.

Will be nice to support it, so the following works

```
    <ProtoFile Include="a.proto" />
    <ProtoFile Include="b.proto" />
```

These are expected to be completly indipendent, and generator will run two times, generating two `.fs` files, so no types are shared

**Comment by enricosada** [comment]
> Just realized generating the files separately, instead of automagically as part of the build might work (and would be preferable) too, but it has another problem: I'd like to include the CLI tool in the same project, but building such a project currently fails unless there are are `ProtoFile` entries.
> 
> EDIT: found `DesignTimeBuild` prop that can be set to false to avoid codegen kicking in.

@et1975 the `DesignTimeBuild` cannot be set in projects, otherwise other .net msbuild sdk behaviour are disabled. If not needed (you dont want to use the integration), do not include `Falanx.Sdk`, but for this original issue, is easier to fix supporting multiple `ProtoFile`s

**Comment by et1975** [comment]
if `DesignTimeBuild` is not an option, the easy fix would be to treat the empty `ProtoFile` as no-op for codegen instead of failing.

**Comment by enricosada** [comment]
> multiple files adds some unforeseen issues like, if both are importing a type thats used in both messages. A better solution would probably be a composite proto files that imports both etc

@7sharp9 i think these are different scenario. often the proto have nothing in common, so we can for example create these in nested namespace, like

```xml
   <ProtoFile Include="a.proto" Namespace="ServiceA" />
   <ProtoFile Include="b.proto" Namespace="ServiceB" />
```

also if these use common types will be duplicated in different namespaces
It's the user who need to choose if create a new proto and import boths, to use shared types.

> if `DesignTimeBuild` is not an option, maybe the easy fix would be to treat the empty `ProtoFile` as no-op for codegen instead of failing?

@et1975 yes, without `<ProtoFile` shouldnt run the generator, not fail the build. I'll check that too

**Comment by enricosada** [comment]
fixed by #98 now multiple `<ProtoFile` are supported.

each one is independent from the others
